
fio --max-jobs=16 --aux-path=/tmp --allow_file_create=0 --significant_figures=10 --name=/dev/vda --filename=/dev/vda --offset=0 --size=268435456 --zonemode=zbd --ioengine=libaio --iodepth=1 --rw=randwrite --bs=16384 --do_verify=1 --verify=md5 --size=67108864 --randseed=1549873713 --thread=1 --direct=1

fio: engine libaio not loadable
fio: failed to load engine
fio: file:ioengines.c:134, func=dlopen, error=libaio: cannot open shared object file: No such file or directory
FAIL
